pgstac_ingestor/config.py

```diff
@@ -11,32 +11,36 @@
 class Settings(BaseSettings):
     dynamodb_table: str
 
-    root_path: Optional[str] = Field(description="Path from where to serve this URL.")
+    root_path: Optional[str] = Field(
+        None, description="Path from where to serve this URL."
+    )
 
     jwks_url: Optional[AnyHttpUrl] = Field(
-        description="URL of JWKS, e.g. https://cognito-idp.{region}.amazonaws.com/{userpool_id}/.well-known/jwks.json"  # noqa
+        None,
+        description="URL of JWKS, e.g. https://cognito-idp.{region}.amazonaws.com/{userpool_id}/.well-known/jwks.json",  # noqa
     )
 
     stac_url: AnyHttpUrl = Field(description="URL of STAC API")
 
-    data_access_role: AwsArn = Field(
+    data_access_role: AwsArn = Field(  # type: ignore
         description="ARN of AWS Role used to validate access to S3 data"
     )
 
     requester_pays: Optional[bool] = Field(
-        description="Path from where to serve this URL.", default=False
+        False,
+        description="Data hosted in Requester-Pays bucket.",
     )
 
     class Config(AwsSsmSourceConfig):
         env_file = ".env"
 
     @classmethod
     def from_ssm(cls, stack: str):
-        return cls(_secrets_dir=f"/{stack}")
+        return cls(_secrets_dir=f"/{stack}")  # type: ignore
 
 
 settings = (
-    Settings()
+    Settings()  # type: ignore
```

Comment:
mypy is not happy but this will get initialized using environment which is why it doesn't need parameters